src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -510,6 +511,10 @@
 						"	set the fake max players count to (online players count + 1)",
 						"	set the shown icon to a random server icon out of {server-icons::*}")
 				.since("2.3");
+		if(Skript.classExists("org.bukkit.event.command.UnknownCommandEvent")) Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "unknown command")
+			.description("Called when a player sends a command that isn't registered")
+			.examples("on unknown command:")
+			.since("INSERT VERSION").requiredPlugins("Paper");
```

Comment:
Still need to replace "insert version" with the current Skript version.

Reply:
What do you mean? Shane said that we should use INSERT VERSION and then they’d make a commit updating it to a specific version, what if it was not 2.5 but 2.5-alpha11? They’d need to do additional things to replace 2.5 with 2.5-alpha11 and it’d be more prone to errors

Reply:
Yeah, they have a script that should replace INSERT VERSION with the correct version anyway, I do not think you need to write an explicit version.

![script contents](https://i.imgur.com/j4Gtsyc.png)

Reply:
It's fine then